### PR TITLE
New response model

### PR DESCRIPTION
**Pull request by sondrev**

### CODE REVIEW

File: search-commons/src/main/java/no/unit/nva/search/resource/response/ResourceSearchResponse.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    private String mainLanguageAbstract;
+
+    @JsonProperty("description")
+    private String description;
+
+    @JsonProperty("alternativeTitles")
+    private List<String> alternativeTitles;
+
+    @JsonProperty("publicationDate")
+    private String publicationDate;
```
Comment (sondrev) [inline search-commons/src/main/java/no/unit/nva/search/resource/response/ResourceSearchResponse.java:33]: @brinxmat This was in the document we looked at together, but not sure if we should have moved it to recordMetadata? Can you double check?

File: search-commons/src/main/java/no/unit/nva/search/resource/response/OtherIdentifiers.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    private List<String> cristin;
+
+    @JsonProperty("scopus")
+    private List<String> handle;
+
+    @JsonProperty("scopus")
+    private List<String> issn;
+
+    @JsonProperty("scopus")
+    private List<String> isbn;
```
Comment (brinxmat) [inline search-commons/src/main/java/no/unit/nva/search/resource/response/OtherIdentifiers.java:21]: Is it all the same?
If Scopus is everything,
Some data is lost!

File: search-commons/src/main/java/no/unit/nva/search/resource/response/ResourceSearchResponse.java (hunk 1 above)
Comment (brinxmat) [reply search-commons/src/main/java/no/unit/nva/search/resource/response/ResourceSearchResponse.java:33]: I guess it is the `{"year": "1232", "month": "12", "day": "02"}` publication date that is the intention

**Review by brinxmat — CHANGES_REQUESTED** [review]
The classes can be records, which might save on annotations too

**Review by ketilaa — APPROVED** [review]

**Comment by ketilaa** [comment]
Godkjente gitt at du vurderer å bruke records. La inn et lite spørsmål rundt publishing details.

File: search-commons/src/main/java/no/unit/nva/search/resource/response/PublishingDetails.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
+package no.unit.nva.search.resource.response;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+class PublishingDetails {
```
[the remark this answers is not in the document]
Comment (brinxmat) [reply search-commons/src/main/java/no/unit/nva/search/resource/response/PublishingDetails.java:5]: serialPublication := (journal, bookSeries, reportSeries, conferenceProceedingsSeries,…)

**Pull request merged**